### DISCUSSION TITLE
LCP is way more than the total time LH took

### DISCUSSION THREAD

**Discussion by thany** [GoogleChrome/lighthouse#16008] [General]
I let LH process a website (local only, can't share it) and it was done doing its thing in about 10 seconds. Fairly normal.

And then it manages to report a LCP of 82 seconds.

So the question is hopefully quite clear. But to be extra verbose: how does it arrive at a LCP that is *much* more than the total time taken to analyse a website? Where is this LCP magiced up? How could it possibly (incorrectly, I believe) know that it's 82 seconds, after being done in 10?

It wasn't a fluke either, it reproduces this weird result every time.
I'm on version 12, and I've done a fresh `npm update -g` as well, just minutes ago.

**Comment by adamraine** [comment]
Lighthouse will use [simulated throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md) by default which loads the page normally and then estimates what performance would look like under certain network and CPU conditions *after*. This means that the performance metrics are not expected to match what you observe locally.

Regardless, 82 seconds does seem pretty high. Can you try using DevTools throttling (i.e. `--throttling-method=devtools`) to see if an LCP that high is repeatable with different throttling strategies?

**Reply by thany** [reply]
I don't even have to. I can see in the "comic" that after the second screenshot everything is loaded and stays the same, so the LCP has already happened by then.

I can see if the throttling method makes for a more realistic measurement, but realistically I'm improving the performance, so it doesn't matter what the number is or means. I guess if I can get 82 seconds down to, say, 30 seconds, it'll be a website as fast as lightning 😀
